src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -4626,12 +4626,9 @@ internal Token NextToken()
                     {
                         if (c1 == '@')
                         {
-                            var c2 = PeekChar();
-                            if (c2 == '{')
-                            {
-                                UngetChar();
-                                return NewToken(TokenKind.At);
-                            }
+                            // We have seen 2 @ characters (generalized splatting), therefore putting the 2nd token back
```

Comment:
I don't think you need to look for `@@` at all - generalized splatting should support `@$var` e.g

Reply:
Ok, I've removed that lookahead now as well although I feel like by removing the lookahead, it slightly increases the risk that special cases of invalid syntax might behave a bit differently now.
If possible, I'd like to implement only 1 feature of generalized splatting in this PR, which is the syntax `get-foo @@{ Param1 = $Param1Value}`. This is the only feature that I need and is useful to me and it is a standalone feature of the generalized splatting RFC.
The `get-foo @hashtableVariable` syntax already enables using a variable for splatting, therefore I do not see the value in adding the `get-foo @$hashtableVariable` syntax and would rather leave it to someone else if they wanted to in the spirit of open source and keeping it minimum viable.